packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -41,333 +32,320 @@ const DEFAULT_QUERY = {
 	per_page: -1,
```

Comment:
Something for later: we should consider removing this line, that said it would probably require some UI/UX changes to handle pagination.

Reply:
I think we need to figure out a way to build a parent-child relationship.